bindings/python/tests/cancellation_timeout_tests.py

```diff
@@ -447,4 +432,4 @@ def txn2(tr):
             if e.code != 1025:
                 raise
 
-    txn2(db)
+    txn1(db)
```

Comment:
The function above needs to be renamed from `txn2` as well

Reply:
Oof. I could have sworn I made that change, but I guess I lost in some local merge conflicts. I'm building libfdb_c right now to run the test, which is why I didn't catch it from, like, running the code.

Reply:
Actually, I did already do it locally, but cleverly forgot to push.

Reply:
Okay, I've now verified that the test runs successfully after removing the other function.